src/coreclr/tools/aot/ILCompiler.RyuJit/ILCompiler.RyuJit.csproj

```diff
@@ -68,6 +68,9 @@
     <Compile Include="..\..\Common\JitInterface\TypeString.cs">
```

Comment:
TypeString.cs is not used by the JIT/EE interface after this change, so it should not be in the JitInterface directory.

The one place that uses it is `OutputInfoBuilder` so it should be next to it.